sample-app/src/components/InputDropdown.tsx

```diff
@@ -97,24 +131,43 @@ export default function InputDropdown({
       evt.preventDefault();
     }
 
+    const isFirstSectionFocused = focusedSectionIndex === 0;
+    const isLastSectionFocused = focusedSectionIndex === options.length - 1;
     const isFirstOptionFocused = focusedOptionIndex === 0;
-    const isLastOptionFocused = focusedOptionIndex === options.length - 1;
+    const isLastOptionFocused = 
+      focusedSectionIndex !== undefined &&
+      focusedOptionIndex === options[focusedSectionIndex].results.length - 1;
     if (evt.key === 'Enter') {
```

Comment:
can we combined these two ifs since we don't do anything specific for just enter now?

Reply:
sure!